docs/INSTALL_MANUAL.md

```diff
@@ -123,18 +178,89 @@ Now that we have downloaded the required files, let's prepare the filesystem.
 
 ```
 # Let's make sure the Scrutiny collector is executable.
-chmod +x /etc/scrutiny/bin/scrutiny-collector-metrics-linux-amd64
+chmod +x /opt/scrutiny/bin/scrutiny-collector-metrics-linux-amd64
+```
+
+### Install Scrutiny Collector systemd service
+
+If you are using systemd, you may configure collector as a systemd service and
+create a timer to invoke it periodically:
+
 ```
+sudo cat > /etc/systemd/system/scrutiny-collector.service <<EOF
+[Unit]
+Description=Scrutiny disk health data collector
+
+[Service]
+Type=idle
+ExecStart=/opt/scrutiny/bin/scrutiny-collector-metrics-linux-amd64 run --api-endpoint "http://localhost:8080"
+TimeoutStartSec=600
+TimeoutStopSec=600
+EOF
+```
+
+Configure collector systemd timer:
+
+```
```

Comment:
Migt make sense to specify `lang` (use ```\sh) and then make it a long list with comments in there like

```sh
# Create this file
sudo cat file <EOF
...
EOF
# Create that
sudo cat file <EOF
...
EOF
# Enable timer
systemctl this
systemctl that
```